BajenaruPavelAlexandru-dir/BajenaruPavelAlexandru.txt

```diff
@@ -0,0 +1,2 @@
+this file was created by Bajenaru Pavel-Alexandru
+
```

Comment:
you do not have the "second commit" line

Reply:
"second commit" is in the second commit "second commit added Hello.cpp"